test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -241,6 +241,52 @@ function prepare-log-file {
 	chown root:root $1
 }
 
+# A helper function for removing salt configuration and comments from a file.
+# This is mainly for preparing a manifest file.
+#
+# $1: Full path of the file to manipulate
+function remove-salt-config-comments {
+	# Remove salt configuration.
+	sed -i "/^[ |\t]*{[#|%]/d" $1
+	# Remove comments.
+	sed -i "/^[ |\t]*#/d" $1
+}
+
+# Prepare the abac policy file from the salt template.
+function setup-abac-authz-policy-file {
```

Comment:
I think that:
- hollowKubelet should have exactly same access level as real "kubelet + kubeproxy"
- controller manager should have exactly the same access level as real controler manager

Again, I didn't follow carefully what was happening in this ares, so @deads2k please correct me if I'm saying something stupid, but I think we should just make our kubemark components behave as the corresponding roles in the real cluster.